network/bridge_networkclient_linux.go

```diff
@@ -1,29 +1,45 @@
 package network
 
 import (
+	"errors"
+	"fmt"
 	"net"
 
 	"github.com/Azure/azure-container-networking/ebtables"
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/Azure/azure-container-networking/netlink"
 	"github.com/Azure/azure-container-networking/network/epcommon"
+	"github.com/Azure/azure-container-networking/network/netlinkinterface"
 )
 
 const (
 	multicastSolicitPrefix = "ff02::1:ff00:0/104"
 )
 
+var errorLinuxBridgeClient = errors.New("LinuxBridgeClient Error")
+
+func newErrorLinuxBridgeClient(errStr string) error {
+	return fmt.Errorf("ErrorLinuxBridgeClient %w : %s", errorLinuxBridgeClient, errStr)
```

Comment:
Perhaps a new type that implements `error` would be better here. This way produces an error message that's a bit odd:
```
ErrorLinuxBridgeClient LinuxBridgeClient Error : <...>
```

Reply:
Noted, will fix in 2nd PR